src/state/gateway.py

```diff
@@ -13,21 +13,20 @@ class GatewayResourceDefinition:
     """Base class containing kubernetes resource definition.
 
     Attrs:
-        namespace: The gateway resource's namespace.
         gateway_name: The gateway resource's name
     """
 
-    namespace: str
+    # We're expecting more fields to be added.
```

Comment:
Would this indicate that we're ignoring a lint error somewhere? If so, I think this could go right above the line in which we're doing the `ignore` :)

Reply:
No I asked thanh to add that comment because using Pydantic for a such simple data object is a bit overkill. Adding that comment let know the reviewer that this class will be extended in a coming PR